migrations/capcons/capabilities.go

```diff
@@ -60,6 +65,38 @@ func (c *AccountCapabilities) Record(
 	)
 }
 
+// ForEachSorted will first sort the capabilities list,
+// and iterates through the sorted list.
+func (c *AccountCapabilities) ForEachSorted(
+	f func(AccountCapability) bool,
+) {
+	c.sort()
+	for _, accountCapability := range c.capabilities {
+		if !f(accountCapability) {
+			return
+		}
+	}
+}
+
+func (c *AccountCapabilities) sort() {
+	c.sortOnce.Do(
```

Comment:
Maybe consider resetting the `sync.Once` in `Record` to avoid surprises when `Record` is called in between two `ForEachSorted` calls.

Reply:
hmm maybe just keeping a boolean flag might be better in that case? resetting `sync.Once` every time `Record` is called is an overkill.